src/main/java/ecdar/controllers/TransitionPaneElementController.java

```diff
@@ -200,8 +197,7 @@ private void expandTransitions() {
      */
     @FXML
     private void refreshTransitions() {
-        SimulatorController.setSelectedTransition(null);
-//        MainController.openReloadSimulationDialog();         // ToDo: Implement

```

Comment:
Er "ToDo: Implement" implementeret af det nye kode?

Reply:
Vi har ikke noget reloadSimulationDialog endnu så den restarter bare simulation. Har også omdøbt metoden til restartSimulation

Reply:
Seems the same, is it not committed?

Reply:
Ups ja har glemt at commit. Jeg tagger dig lige når det er færdigt så må du meget gerne kigge det igennem igen 😄

Reply:
ved ik hvorfor github ikke skriver "outdated" men den er altså ændret. Det ligger på linje 177 nu